internal/graph/check.go

```diff
@@ -204,8 +208,8 @@ type directDispatch struct {
 }
 
 func (cc *ConcurrentChecker) checkDirect(ctx context.Context, crc currentRequestContext, relation *core.Relation) CheckResult {
-	log.Ctx(ctx).Trace().Object("direct", crc.parentReq).Send()
```

Comment:
I'd keep this trace log for people not using tracing, but just `--log-level=trace`

Reply:
My bad